.github/workflows/stale.yaml

```diff
@@ -0,0 +1,47 @@
+name: Mark and Close Stale Issues and PRs
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # runs every day at midnight UTC
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Mark and close stale issues and PRs
+        uses: actions/stale@v9
```

Comment:
Please fix the commit id. 
`actions/stale@5bef64f19d7facfb25b37b414482c7164d639639 #v9.1.0`